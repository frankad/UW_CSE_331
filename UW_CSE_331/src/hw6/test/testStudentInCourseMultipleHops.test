# Find a path with multiple hops from one student to an other students 
# using studentInCourse full dataset

# build graph
LoadGraph studentInCourse studentInCourse.tsv

# test the path
FindPath studentInCourse Olivia	Abigal