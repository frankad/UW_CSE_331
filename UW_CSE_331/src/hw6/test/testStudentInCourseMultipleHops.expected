# Find a path with multiple hops from one student to an other students 
# using studentInCourse full dataset

# build graph
loaded graph studentInCourse

# test the path
path from Olivia to Abigal:
Olivia to Chris via CSE331
Chris to Sam via EE474
Sam to Liya via EE448
Liya to Sophia via MATH390
Sophia to Abigal via ECON201